py/damas.py

```diff
@@ -50,12 +50,7 @@ def create( self, keys ) :
         headers.update(self.headers)
         r = requests.post(self.serverURL+"/api/create/", data=json.dumps(keys),
             headers=headers, verify=False)
-        if r.status_code == 201 or r.status_code == 207:
-            return json.loads(r.text)
-
-        error_obj = ErrorHandler(str(r.status_code))
-        error_response = error_obj.handler("/api/create/")
-        return error_response
+        return r.status_code, r.text
```

Comment:
the r.text String would still need to be converted to the correct type returned by the server